surveys/2024-04/questions.md

```diff
@@ -0,0 +1,142 @@
+    - Direct end users: Users who run tools themselves
+    - Application operators: Users who interacts with existing running applications
+    - Application developers: Frontend tools consumers, backend server authors, tools authors
+    - Library & package authors: Users who write libraries and packages to be included on other applications
+    - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+    - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+
+- Are you part of a group not covered ? If yes which one ? (open question)
+
+- What is your use cases of Node.js ? (if multiple use case, separate with a comma) (open question)
+
+## What Node.js binaries do you use
+
+- What is your Operating System for your local development environment ? (single choice - Other should give an input)
+	- Linux
+	- Windows
+	- MacOs
+	- Other?
+
+- What is your Operating System in which you are running Node.js in production ? Do you plan to change that in the near future? (single choice - Other should give an input)
+	- Linux/Unix
+	- Windows
+	- MacOs
+	- Other?
+
+- What is your architecture in which you are running Node.js with Node.js ? Do you plan to change that in the near future? (single choice - Other should give an input)
+    - x64
+    - ia32
+    - arm
+    - ppc
+    - s390
+    - Other ?
+
+- How do you get your Node.js executables? (multiple choice)
+    - With a Node.js version manager: nvm, n, nave, nvs, volta, etc.
+    - Using the official installer
+    - Using package managers like npm or yarn and install Node.js as a package
+    - Using a system package manager: apt-get, brew, dnf, yum …
+    - Downloading binaries directly from Nodejs.org
+    - Building Node.js from Source
+    - Other
+
+- What package manager do you use ? (multiple choice - Other should give an input)
+    - npm
+    - yarn
+    - cnpm
+    - pnpm
+    - Other ? 
+
+- If you use a version manager which one do you use (multiple choice - Other should give an input)
+    - nvm
+    - n
+    - nave
+    - nvs
+    - volta
+    - Other ?
+
+## Project Priorities and Direction
+
+- Are you aware of the defined Node.js [Technical Priorities](https://github.com/nodejs/node/blob/main/doc/contributing/technical-priorities.md)? (single choice)
+    - Yes
+    - No
+
+- Which of the current technical priorities are important to you ? (multiple choice)
+    - Modern HTTP
+    - Suitable types for end-users
+    - Documentation
+    - WebAssembly
+    - ESM
+    - Support for features from the latest ECMAScript spec
+    - Observability
+    - Permissions/policies/security model
+    - Better multithreaded support
+    - Single Executable Applications
+
+- Are there technical priorities that you believe are missing (open question)
+
+- What is important to you ? (multiple choice)
+    - Good understanding of the direction of the project
```

Comment:
There are too many options, we should trim it down a bit to include no more than 10, or split it into different questions by topic, otherwise I think people would just get tired and skip.